src/components/Toolbelt/Toolbelt.test.js

```diff
@@ -1,39 +1,82 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount, render } from 'enzyme'
 
 import { fieldMode } from '../../enums'
+import tools from '../../data/tools'
 
 import { Toolbelt } from './Toolbelt'
 
-let component
-
-beforeEach(() => {
-  component = shallow(
-    <Toolbelt
-      {...{
-        fieldMode: fieldMode.OBSERVE,
-        handleFieldModeSelect: () => {},
-      }}
-    />
-  )
-})
+const noop = () => {}
 
-test('renders shop inventory', () => {
-  expect(component).toHaveLength(1)
-})
+// https://stackoverflow.com/questions/58070996/how-to-fix-the-warning-uselayouteffect-does-nothing-on-the-server
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useLayoutEffect: jest.requireActual('react').useEffect,
+}))
```

Comment:
i think this will all go away with testing library. pretty sure this is weirdness with enzyme and material ui :/ (or maybe it was from the version mismatches.. either way, expecting it to go away)